waiter/src/waiter/core.clj

```diff
@@ -694,7 +676,7 @@
                          (let [local-router (InetAddress/getLocalHost)
                                waiter-router-hostname (.getCanonicalHostName local-router)
                                waiter-router-ip (.getHostAddress local-router)]
-                           (waiter-request?-factory #{hostname waiter-router-hostname waiter-router-ip})))})
+                           (waiter-request?-factory (set [hostname waiter-router-hostname waiter-router-ip]))))})
```

Comment:
`waiter-router-hostname` and `waiter-router-ip` are the same value when I run locally and this causes an error with the set literal.